pkg/manager/member/pd_scaler.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package member
+
+import (
+	"fmt"
+
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap.com/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/controller"
+	apps "k8s.io/api/apps/v1beta1"
+)
+
+type pdScaler struct {
+	pdControl controller.PDControlInterface
+}
+
+// NewPDScaler returns a *pdScaler
+func NewPDScaler(pdControl controller.PDControlInterface) *pdScaler {
```

Comment:
It's better to return a `Scaler` for exported function. So we don't have to add type assertion `var _ Scaler = &pdScaler{}`